catroid/src/org/catrobat/catroid/ui/adapter/BrickAdapter.java

```diff
@@ -475,6 +495,47 @@ private void moveUserBrick(int from, int to) {
 		script.addBrick(getPositionInUserScript(to), brick);
 	}
 
+	private void moveNestingBrick(int from, int to) {
+		Sprite currentSprite = ProjectManager.getInstance().getCurrentSprite();
+
+		int[] tempFrom = getScriptAndBrickIndexFromProject(from);
+		int scriptPositionFrom = tempFrom[0];
+		int brickPositionFrom = tempFrom[1];
+
+		Script fromScript = currentSprite.getScript(scriptPositionFrom);
+		Brick brick = fromScript.getBrick(brickPositionFrom);
+
+		NestingBrick nestingBrick = (NestingBrick) brick;
+		List<NestingBrick> nestingBricks = nestingBrick.getAllNestingBrickParts(true);
+		NestingBrick endNestingBrick = nestingBricks.get(nestingBricks.size() - 1);
+		List<Brick> fromScriptBrickList = fromScript.getBrickList();
+		int endPosition = fromScriptBrickList.indexOf(endNestingBrick);
+		int count = endPosition - brickPositionFrom;
+		boolean isNewPositionBetweenStartAndEndNestedBrick = to >  from && count > to - from;
```

Comment:
please remove the unnecessary space after "to > " there should be only one. (There is one more instance of the same problem -> just have a look at the formatters suggestions.